training/src/main/java/wolox/training/models/Privilege.java

```diff
@@ -0,0 +1,56 @@
+package wolox.training.models;
+
+import com.google.common.base.Preconditions;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import org.thymeleaf.util.StringUtils;
+import wolox.training.utils.MessageConstants;
+
+@Entity
+public class Privilege {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
I just realize that I forgot to tell you something before! Sorry! 
When we use PostgreSQL as our database, I think that if we use @GeneratedValue(strategy = GenerationType.AUTO), that strategy will maintain an autoincrement id shared with every table (I mean, not one per table). You can check it out! This doesn't happen in MySQL, there this strategy works just fine!
Anyway, there is no need to change everything now but it's important for you to know about that :) The correct way to generate the ID for PostgreSQL is using SEQUENCE. For example, for Book it would be something like this:
```
  @Id
  @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "BOOK_SEQ")
  @SequenceGenerator(name = "BOOK_SEQ", sequenceName = "book_id_seq", allocationSize = 1)
  private Long id;
```
So, PostgreSQL will create a sequence for each table